docs/how-to/h-manage-units.md

```diff
@@ -65,34 +71,32 @@ be obtained using the `get-admin-credentials` action output:
 BOOTSTRAP_SERVERS=$(juju run kafka-k8s/leader get-admin-credentials | grep "bootstrap.servers" | cut -d "=" -f 2)
 ```
 
-Admin client authentication information is stored in the 
-`/etc/kafka/client.properties` file that is present on every Apache Kafka
-container. The content of the file can be accessed using `juju ssh` command:
+Admin client authentication information is stored in the `/etc/kafka/client.properties` file that is present on every Apache Kafka container. 
+The content of the file can be accessed using `juju ssh` command:
 
 ```
 juju ssh --container kafka kafka-k8s/leader 'cat /etc/kafka/client.properties'
 ```
 
-where `--container kafka` is to select the Apache Kafka workload container of the unit. By default, the charm operator container is opened.
+where the `--container kafka` is to select the Apache Kafka workload container of the unit. By default, the charm operator container is opened.
 
 This file can be provided to the Apache Kafka bin commands via the `--command-config`
-argument. Note that `client.properties` may also refer to other files (
-e.g. truststore and keystore for TLS-enabled connections). Those
-files also need to be accessible and correctly specified. 
+argument. Note that `client.properties` may also refer to other files (e.g. truststore and keystore for TLS-enabled connections). 
+Those files also need to be accessible and correctly specified. 
 
-Commands can be run within a Apache Kafka broker, since both the authentication 
+Commands can be run within an Apache Kafka broker, since both the authentication 
 file (along with the truststore if needed) and the Charmed Apache Kafka binaries are 
 already present. 
 
-#### Example (listing topics)
+#### Listing topics example
 
-For instance, to list the current topics on the Apache Kafka cluster, you can run:
+For instance, to list the current topics on the Apache Kafka cluster, run:
```

Comment:
Does this sound better? It jumped out at me to start with that wording
```suggestion
To list the current topics on the Apache Kafka cluster, run:
```

Reply:
You're right, the command looks like it's useful in the most general case, not just in some specific example. Let's also update this section's heading, to make it all a bit more generic.